### PR TITLE
Chore – Marker-genes-only search

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
Companion PR of https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/347.

Manual checks

- [x] This branch is passing the CI tests as part of a pull request in the atlas-web-bulk or atlas-web-scxa repos and I have indicated that PR here.

### CODE REVIEW

File: src/test/java/uk/ac/ebi/atlas/resource/DataFileHubIT.java [hunk 1]
```diff
@@ -12,100 +11,100 @@
...
+    }
+
+    @Test
+    void findsMarkerGeneFiles() {
+        var experimentAccession = jdbcUtils.fetchRandomExperimentAccession();
+        LOGGER.info("Test marker gene files for experiment {}", experimentAccession);
+        assertAtlasResourceExists(subject.getSingleCellExperimentFiles(experimentAccession).markerGeneTsvs.values());
+    }
+
+// TODO Rethink this test since not all experiments have inferred cell type annotations
```
Comment (ke4) [inline src/test/java/uk/ac/ebi/atlas/resource/DataFileHubIT.java:71]: Shall we have a `fetchRandomExperimentAccessionForInferredCellTypes()` method and that would only return a random accession from a set of experiment accessions that HAS inferred cell type annotations?

Comment (ke4) [inline src/test/java/uk/ac/ebi/atlas/resource/DataFileHubIT.java]: Shall we have a fetchRandomExperimentAccessionForInferredCellTypes() method and that would only return a random accession from a set of experiment accessions that HAS inferred cell type annotations?
That way we could enable this test...

**Review by ke4 — CHANGES_REQUESTED** [review]
Look great! Just a small comment to be able to enable a test.
Thanks for your work 👍

File: src/test/java/uk/ac/ebi/atlas/resource/DataFileHubIT.java (hunk 1 above)
Comment (alfonsomunozpomer) [reply src/test/java/uk/ac/ebi/atlas/resource/DataFileHubIT.java]: I moved this test to `DataFileHubIT` in `/app` and added the method as we discussed. However, read my comments in https://github.com/ebi-gene-expression-group/atlas-web-single-cell/pull/348 regarding `DataFileHub`.

**Review by ke4 — APPROVED** [review]
Looks fine by me 👍

**Pull request merged**